packages/db-migrations/databases/fxa/patches/patch-150-151.sql

```diff
@@ -0,0 +1,9 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('150');
+
+ALTER TABLE `linkedAccounts`
+  ADD UNIQUE INDEX id_providerId_idx (`id`, `providerId`),
```

Comment:
@vbudhram can you confirm whether this is unique?

Reply:
We should be good to go, https://github.com/mozilla/fxa/blob/643c12606e38877ece290a1f9dc063eccce8584b/packages/db-migrations/databases/fxa/patches/patch-123-124.sql#L7